src/containers/Header.jsx

```diff
@@ -25,9 +38,12 @@ export class Header extends React.Component {
   }
 
   handleChange = (event, value) => {
-    const { history } = this.props;
+    const { history, resetSearchMovies, searchQuery } = this.props;
     history.push(tabs[value].PATH);
     this.setState({ value });
+    if (searchQuery) {
```

Comment:
What about when `searchQuery` is empty? You should reset the page to `1`, right?